README.md

```diff
@@ -1,752 +1,311 @@
-<img src="static/logo.png" width="400" />
-
-<br/>
+# Netlify Build
 
 [![Coverage Status](https://codecov.io/gh/netlify/build/branch/master/graph/badge.svg)](https://codecov.io/gh/netlify/build)
 [![Build](https://github.com/netlify/build/workflows/Build/badge.svg)](https://github.com/netlify/build/actions)
 
-# Netlify Build
-
-Netlify build is the next generation of CI/CD tooling for modern web applications.
 
-[Sign up for the private beta](https://www.netlify.com/build/plugins-beta/)
+Netlify Build is a new, pluggable tool for running builds locally and in Netlify CI. It introduces Build Plugins, which are **now in public beta.** Learn how to enable your site to use Netlify Build and Build Plugins in the [Netlify docs](https://docs.netlify.com/configure-builds/plugins).
```

Comment:
[question / sand] At first I thought "Netlify CI" was a typo of "Netlify CLI" but after a few re-reads I think this is probably not a typo but rather referring to the same concept that a later mention of "production CI" is.  

So my question is - is this a typo of CLI? 

If it's not a typo my sand is - I worry that other people might think this is a typo of CLI and get confused. Perhaps something like "Netlify's production CI" would be more clear as well as tie into the later mention of "production CI"